docs/intro/install.rst

```diff
@@ -69,20 +69,25 @@ Overview
 Upload Elgg
 -----------
 
--  Download the `latest version of Elgg`_
--  Upload the ZIP file with an FTP client to your server
--  Unzip the files in your domain's document root (/home/username/www).
-
-.. _latest version of Elgg: http://elgg.org/download.php
-
-Elgg also supports installing from source with Composer:
+With Composer (recommended):
 
 .. code:: shell
 
     cd /path/to/wwwroot/
     composer self-update
     composer global require "fxp/composer-asset-plugin:~1.0"
-    composer create-project Elgg/Elgg:~2.0 .
+    composer create-project elgg/starter-project:dev-master .
+
+If you're not comfortable with Composer, you can install from a pre-packaged zip:
```

Comment:
Of course, I read it. That's why I said that recommending the composer install and listing the "old way" as alternative only is in my opinion very likely not good with non-developers / inexperienced users in mind.

The commands listed for installing via composer alone will result in an inexperienced user being complete lost (let alone it will not work if composer is not installed). The advantages of composer might be great for developing and the instructions seem to have been written from the perspective of a developer addressing another developer. But I just don't see it working for non-developers.